src/Screens/App/Screens/BooksDashboard/Screens/SearchBar/Screens/FilterSelector/style.css

```diff
@@ -0,0 +1,36 @@
+.filter-container {
+  margin: 0 28px 0 35px;
+  position: relative;
+  display: block;
+  width: 240px;
+  height: 40px;
+  background: #f4f4f4;
+  overflow: hidden;
+}
+
+.filter-container::after {
+  content: '\25BC';
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  padding: 11px 15px;
+  background: #f4f4f4;
```

Comment:
To uppercase

Reply:
Extract common properties along with these appearing in `.filter-container` and `.filter-type-select`